flexmeasures/data/models/planning/tests/test_solver.py

```diff
@@ -792,3 +792,75 @@ def test_validate_constraints(
     )
 
     assert set(expected_constraint_type_violations) == constraint_type_violations_output
+
+
+def test_infeasible_problem_error(add_battery_assets):
+    """Try to create a schedule with infeasible constraints. soc-max is 4.5 and soc-target is 8.0"""
+
+    # get the sensors from the database
+    _epex_da, battery = get_sensors_from_db()
+
+    # time parameters
+    _tz, start, end, resolution = get_time_parameters(
+        [2015, 1, 2], [2015, 1, 3], {"hours": 1}
+    )
+
+    def compute_schedule(flex_model):
+        scheduler = StorageScheduler(
+            battery,
+            start,
+            end,
+            resolution,
+            flex_model=flex_model,
+        )
+        schedule = scheduler.compute()
+
+        soc_schedule = integrate_time_series(
+            schedule,
+            soc_at_start,
+            decimal_precision=1,
+        )
+
+        return soc_schedule
+
+    # soc parameters
+    soc_at_start = battery.get_attribute("soc_in_mwh")
+    soc_min = 0.5
+    soc_max = 4.5
+    soc_maxima = [{"datetime": "2015-01-02T15:00:00+01:00", "value": 1.0}]
```

Comment:
I think we can take out `soc_maxima` and `soc_minima` as we are already creating a infeasible condition with the target. This simplifies the test.

Reply:
put the soc min and max into the flexmodel and removed the soc minima and soc maxima.

Thanks for reviewing!